src/Test/Utilities/Shared/Diagnostics/DiagnosticExtensions.cs

```diff
@@ -97,6 +97,24 @@ public static TCompilation VerifyDiagnostics<TCompilation>(this TCompilation c,
         {
             var diagnostics = c.GetDiagnostics();
             diagnostics.Verify(expected);
+
+            var unreferencedAssemblyDiagnosticIds = c.UnreferencedAssemblyIdentityDiagnosticCodes;
```

Comment:
The expected identity values must be verified as well.

---
In reply to: [89177753](https://github.com/dotnet/roslyn/pull/15436#discussion_r89177753) [](ancestors = 89177753)